src/libtorch.cc

```diff
@@ -350,9 +351,11 @@ ModelState::ParseParameters()
         TRITONSERVER_ErrorDelete(err);
       }
     } else {
+      // Override, disable NvFuser till fixed
+      enable_nvfuser = false;
```

Comment:
I think if the issue is in 22.03 pytorch and the default value for NVFuser is already false, we should not be forcing false here. Instead, we can add a release note for using NVFuser in Pytorch backend and mention that it doesn't work because of an issue in Pytorch.

Reply:
The issue is that many users will update the Triton container version they use, then see a failure and open a bug. 

Only for us to then point them to the release notes. I rather will avoid any cases where a known error can cause issues in model loading / starting Triton server

Reply:
Ok, I'll be resolving this thread.